packages/react-ui/src/widgets/CategoryWidgetUI/CategoryWidgetUI.js

```diff
@@ -517,15 +536,14 @@ function CategoryWidgetUI(props) {
       )}
       <CategoriesWrapper container item>
         {animValues.length ? (
-          animValues.map((d, i) => (
-            <CategoryItem
-              key={i}
-              data={d}
-              onCategoryClick={() =>
-                showAll ? handleCategoryBlocked(d.name) : handleCategorySelected(d.name)
-              }
-            />
-          ))
+          <FixedSizeList
+            height={320}
+            width='100%'
+            itemCount={animValues.length}
+            itemSize={48}
+          >
+            {Row}
+          </FixedSizeList>
```

Comment:
This is what is being recommended by MUI: https://mui.com/material-ui/react-list/#virtualized-list